### PR TITLE
Bug 1311457 - Show termination time in cluster detail page instead of timedelta

### PR DESCRIPTION
**Pull request by jezdez**
This also shows a tooltip for the timedelta and the current UTC time as a tooltip over the time in the footer

### CODE REVIEW

**Comment by jezdez** [comment]
screen shot 2016-10-21 at 15 24 37
screen shot 2016-10-21 at 15 24 30

**Comment by jezdez** [comment]
@maurodoglio r?

File: atmo/static/js/refresher.js [hunk 1]
```diff
@@ -5,8 +5,17 @@ $(function() {
...
           refresh_timer = '#refresh-timer',
           refresher = $('#refresher'),
+          time = $('#time'),
           timeout = 60,
           timeout_id;
+      var utc_now = function() {
+        return moment().utcOffset(0).format('YYYY-MM-DD HH:mm:ss');
+      }
+      var updateTime = function() {
+        time.attr('data-original-title', utc_now());
```
Comment (maurodoglio) [inline atmo/static/js/refresher.js]: Where is data-original-title used? I feel like I'm missing something.

Comment (jezdez) [inline atmo/static/js/refresher.js]: That's the magic attribute title of the tooltip plugin: http://stackoverflow.com/a/27237034/153296

**Pull request merged**